train.lua

```diff
@@ -16,6 +16,7 @@ cmd:option('--minLR', 0.00001, 'minimum learning rate')
 cmd:option('--saturateEpoch', 20, 'epoch at which linear decayed LR will reach minLR')
 cmd:option('--maxEpoch', 50, 'maximum number of epochs to run')
 cmd:option('--batchSize', 10, 'mini-batch size')
+cmd:option('--gpu', 0, 'Zero-indexed ID of the GPU to use; for CPU mode set --gpu = -1')
```

Comment:
Sorry, I used the code from another project and left the comment unchanged. The -1 has absolutely no effect, in fact the number is only important if you want to use GPU.

The zero-index thing is just because I like to count from zero 😁